internal/conf/config.go

```diff
@@ -22,6 +22,7 @@ var DefaultOperatorConfig = OperatorConfig{
 	SmbServicePort:            445,
 	SmbdPort:                  445,
 	MetricsExporterMode:       "disabled",
+	ImagePullPolicy:           "ifNotPresent",
```

Comment:
As per [docs](https://pkg.go.dev/k8s.io/api@v0.24.3/core/v1#PullPolicy) isn't it correct to use `IfNotPresent` and not `ifNotPresent`?

Reply:
@anoopcs9 Good catch! thanks, will fix immediately.